consensus/src/persistent_liveness_storage.rs

```diff
@@ -373,6 +373,7 @@ impl<T: Payload> PersistentLivenessStorage<T> for StorageWriteProxy {
             highest_timeout_certificate,
         ) {
             Ok(mut initial_data) => {
+                // @REVIEW: Why do we have to clean these things up at startup?
```

Comment:
we don't retry or panic if we fail to prune it normally, so there may be some garbage (dangling blocks) in storage, and we want to make sure it won't grow infinitely

Reply:
I see... part of this is that we don't take the approach "remove all blocks prior to round X", so we have to ensure everything gets GC'd.

To be defensive, it could be worth considering something along those lines: e.g., prefix this data with the epoch and blow away all data for an epoch once its over

Reply:
I feel this is sufficient for now, it things get bad, oncall probably would have to do a manual GC